src/imagej/convert.py

```diff
@@ -458,6 +458,75 @@ def supports_imglabeling_to_labeling(obj):
     return isinstance(obj, jc.ImgLabeling)
 
 
+#######################
+# IndexImg converters #
+#######################
+
+
+def index_img_to_roi_tree(
```

Comment:
I read up on SRP quickly and I'm not too sure either. Potentially I could move these conversions over to a new `roi.py` module that I intend to introduce later for ROI related operations. What do you think about that?

Reply:
Well, regardless of moving this function, you'll still be doing two different things in the function:
1) Convert the index image to python if it is Java
2) Create the RoiTree from the NumPy array

Like I said, I'm not sure that it's enough of a SRP violation that you need to separate these things.

Reply:
@gselzer Are you thinking it could make sense to keep this function contract, but split it into two private helper functions called sequentially? If so, it can always be done later, in case we find other scenarios that would benefit from calling one half or the other of the function.

Reply:
@ctrueden not really, I'm not sure I even want a change here, I was just noting that this function really does two separate functions, and thought it was a bit strange but at the same time said that I liked the effect.

Reply:
This is a great discussion and I learned about SRP! Thanks @gselzer ! I'll keep that in mind going forward. I looked at this for a bit and right now I want to leave it as it is. It works and is relatively simple to extract out if we have some future scenario that calls for it. I'm going to mark this resolved.